packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -555,27 +558,25 @@ String? _fileNameFromUri(String? uri) {
 class VmObjectDisplayBasicLayout extends StatelessWidget {
   const VmObjectDisplayBasicLayout({
     required this.object,
-    this.generalInfoTitle = 'General Information',
     required this.generalDataRows,
-    this.sideCardTitle = 'Object Details',
     this.sideCardDataRows,
+    this.generalInfoTitle = 'General Information',
+    this.sideCardTitle = 'Object Details',
     this.expandableWidgets,
   });
 
   final VmObject object;
+  final List<MapEntry<String, Widget Function(BuildContext)>> generalDataRows;
```

Comment:
Nit: it might be worth doing something like `typedef Widget WidgetBuilder(BuildContext)` and using `WidgetBuilder` instead of the full signature. @kenzieschmoll, thoughts?

Reply:
Changed to Flutter's WidgetBuilder typedef